.github/workflows/recursion.yml

```diff
@@ -37,7 +37,14 @@ jobs:
         run: |
           ./build.sh
         working-directory: recursion/circuits
-
+        
+      - name: Install npm-check-updates and upgrade dependencies
+        if: github.event_name == 'schedule'
+        working-directory: recursion/js
+        run: |
+          npm install -g npm-check-updates
+          ncu -u
```

Comment:
Will this update all packages to latest? Can we only update the relevant noir / bb packages?